fe/fe-core/src/main/java/com/starrocks/qe/DefaultCoordinator.java

```diff
@@ -910,7 +910,8 @@ public RowBatch getNext() throws Exception {
                         copyStatus.getErrorMsg().equals(FeConstants.BACKEND_NODE_NOT_FOUND_ERROR)) {
                     ec = InternalErrorCode.CANCEL_NODE_NOT_ALIVE_ERR;
                 } else if (copyStatus.isTimeout()) {
-                    ErrorReport.reportTimeoutException(ErrorCode.ERR_QUERY_TIMEOUT, errMsg);
+                    ErrorReport.reportTimeoutException(
+                            ErrorCode.ERR_TIMEOUT, "Query", jobSpec.getQueryOptions().query_timeout, errMsg);
```

Comment:
Just query_timeout or depending on isExecLoadType()?

Reply:
Only query is used here, 
and `TQueryOptions.query_timeout` is used uniformly for query and insert.